doc/README

```diff
@@ -6,8 +6,11 @@ This directory contains source for the man pages for the NVM Library.
 If you're looking for documentation to get you started using NVML,
 start here: http://pmem.io/nvml and follow the links to examples and
 man pages.  Developers new to NVML are probably looking for libpmemobj.
-To generate web-based documentation, you need to have groff and doxygen
-installed.
+To generate web-based documentation, you need to have groff, doxygen, pandoc
+and pp installed. PP generic preprocessor you can find at
```

Comment:
You can find PP generic processor at ...
or
PP generic preprocessor can be found at